### ISSUE TITLE
[iOS] Enable Swift module stability on JitsiMeet binary framework

### ISSUE THREAD

**Issue by SBiOSoftWhare** [jitsi/jitsi-meet#4812]
Hi, 

Here is some information to fix the build issue on Xcode 11.2 reported at #4807.

In Swift 5.1 Apple introduced module stability for binary frameworks to allow forward-compatibility of a library. Like  explained on [Swift website](https://swift.org/blog/abi-stability-and-more/) this means:

> As an example, you could build a framework using Swift 6, and that framework’s interface would be readable by both Swift 6 and a future Swift 7 compiler.

This prevent having errors such as: `Module compiled with Swift 5.1 cannot be imported by the Swift 5.1.2 compiler`.

To enable this feature and avoid recompiling the library for each new Swift version you have to enable `BUILD_LIBRARY_FOR_DISTRIBUTION` in build settings of the project.

<img width="1261" alt="module_stability" src="https://user-images.githubusercontent.com/2205780/68126727-b275bc80-ff14-11e9-8769-817b3619c201.png">

You can find more informations about this in WWDC session [Binary Frameworks in Swift](https://developer.apple.com/videos/play/wwdc2019/416/).

**Comment by saghul** [comment]
Thanks for the heads up! We’ll make sure to turn that on before the next SDK release goes out.

Is that available in Xcode 11.1 already or is it a Xcode 11.2 feature?

**Comment by SBiOSoftWhare** [comment]
You're welcome 🙂

> Is that available in Xcode 11.1 already or is it a Xcode 11.2 feature?

This should be available since Xcode 11.0.

**Comment by saghul** [comment]
Looks like that option doesn't work together with the legacy build system:

```
14:37:22 === BUILD TARGET JitsiMeet OF PROJECT sdk WITH CONFIGURATION Release ===
14:37:22 The legacy build system does not support building projects with Swift when SWIFT_ENABLE_LIBRARY_EVOLUTION is enabled.
14:37:22 The legacy build system does not support building projects with Swift when SWIFT_ENABLE_LIBRARY_EVOLUTION is enabled.
14:37:22 ** ARCHIVE FAILED **
```

I'll get back to this as time allows, but alas, it's not a 1 line change.
